deploy/helm/events-collector-config.yaml

```diff
@@ -93,41 +93,81 @@ processors:
           - delete_key(resource.attributes, "sw.k8s.cronjob.found")
           - delete_key(resource.attributes, "sw.k8s.node.found")
 
+  transform/stringify_body:
+    error_mode: ignore
+    log_statements:
+      - context: log
+        statements:
+          # convert body to string
+          - set(body, body.string)
+
 {{- if .Values.otel.manifests.enabled }}
   transform/set_labels_and_annotations_for_entities:
     error_mode: ignore
     log_statements:
       - context: log
         statements:
-          # set entity annotations for the resource
-          - set(attributes["k8s.pod.annotations"], body["metadata"]["annotations"]) where body["kind"] == "Pod"
-          - set(attributes["k8s.deployment.annotations"], body["metadata"]["annotations"]) where body["kind"] == "Deployment"
-          - set(attributes["k8s.statefulset.annotations"], body["metadata"]["annotations"]) where body["kind"] == "StatefulSet"
-          - set(attributes["k8s.replicaset.annotations"], body["metadata"]["annotations"]) where body["kind"] == "ReplicaSet"
-          - set(attributes["k8s.daemonset.annotations"], body["metadata"]["annotations"]) where body["kind"] == "DaemonSet"
-          - set(attributes["k8s.job.annotations"], body["metadata"]["annotations"]) where body["kind"] == "Job"
-          - set(attributes["k8s.cronjob.annotations"], body["metadata"]["annotations"]) where body["kind"] == "CronJob"
-          - set(attributes["k8s.node.annotations"], body["metadata"]["annotations"]) where body["kind"] == "Node"
-          - set(attributes["k8s.service.annotations"], body["metadata"]["annotations"]) where body["kind"] == "Service"
-          - set(attributes["k8s.persistentvolume.annotations"], body["metadata"]["annotations"]) where body["kind"] == "PersistentVolume"
-          - set(attributes["k8s.persistentvolumeclaim.annotations"], body["metadata"]["annotations"]) where body["kind"] == "PersistentVolumeClaim"
-          - set(attributes["k8s.ingress.annotations"], body["metadata"]["annotations"]) where body["kind"] == "Ingress"
-          - set(attributes["k8s.configmap.annotations"], body["metadata"]["annotations"]) where body["kind"] == "ConfigMap"
+          # assign entity prefix k8s.entity.annotations.*
+          - flatten(body["metadata"]["annotations"], "k8s.pod.annotations") where body["kind"] == "Pod"
```

Comment:
These operations modify the original manifests. That should not happen. You may need to do this on a copy of the original collection.